proxy/http/HttpTunnel.cc

```diff
@@ -589,8 +589,8 @@ HttpTunnel::kill_tunnel()
     ink_assert(producer.alive == false);
   }
   active = false;
-  this->deallocate_buffers();
   this->deallocate_redirect_postdata_buffers();
+  this->deallocate_buffers();
```

Comment:
@oknet If we should always call {{deallocate_redirect_postdata_buffers()}} before {{deallocate_buffers()}} then should the {{HttpTunnel}} class handle this itself.